docs/en/Mounting-Alluxio-FS-with-FUSE.md

```diff
@@ -103,6 +105,34 @@ same way you would for any other client application.
 One possibility, for example, is to edit `$ALLUXIO_HOME/conf/alluxio-site.properties` and set your
 specific alluxio client options. Note that these changes should be before Alluxio-FUSE starts.
 
+### Configure mount point options
+
+By default, Alluxio Fuse mount point could only be accessed by he user mounting the Alluxio.
```

Comment:
"could only" -> "can only"
"he user" -> "the user"
"mounting the Alluxio" -> "mounting the Alluxio namespace to the local filesystem"

Reply:
Done, thanks